app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -36,6 +36,14 @@ export const isOperandSearch = word => {
 
 const containsInArray = (word, array) => array.some(operand => word.includes(operand))
 
+export const canCloseColumn = projectBoard =>
+  Object.keys(projectBoard.visibleColumns).reduce((sum, iteration) =>
```

Comment:
You can use [values](https://underscorejs.org/#values) here.

```js
export const canCloseColumn = projectBoard => 
  values(projectBoard.visibleColumns).filter(Boolean).length > 1
```

Reply:
Nice!!! Thanks!!!